### PR TITLE
Timed psql tls

### PR DESCRIPTION
**Pull request by michaelimfeld**

### CODE REVIEW

File: charts/timed/values.yaml [hunk 1]
```diff
@@ -231,6 +231,25 @@ postgresql:
...
+    certificate:
+      # postgresql.tls.certificate.name -- Name of certificate
+      name: psql-server
+      # postgresql.tls.certificate.dnsNames -- DNS names of certificate
+      dnsNames: []
+      # postgresql.tls.certificate.issuerRef -- Issuer ref
+      issuerRef:
+        group: cert-manager.io
+        kind: ClusterIssuer
+        name: letsencrypt-prod
```
Comment (hairmare) [inline charts/timed/values.yaml:252]: I'd like to see issuerRef configurable so we can also support other issuers.

Also, please add a default doc snippet so helm-docs doesn't dump json to the docs.

File: charts/timed/Chart.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: timed
 description: Chart for Timed application
 type: application
-version: 0.3.20
+version: 0.3.21
```
Comment (tongpu) [inline charts/timed/Chart.yaml:5]: Because you're introducing a new feature you should bump the minor release.

Comment (hairmare) [reply charts/timed/Chart.yaml:5]: Strictly speaking SemVer only mandates such a bump after reaching the 1.x version. In this case it makes sense to bump minor anyhow ;)

File: charts/timed/values.yaml [hunk 3]
```diff
@@ -221,7 +221,7 @@ postgresql:
   # postgresqlPassword: s3cr3t
   image:
     # postgresql.image.tag -- PostgreSQL image version to use
-    tag: 12.2.0
+    tag: 12.3.0
```
Comment (hairmare) [inline charts/timed/values.yaml:224]: Can you add a quick comment as to why we are overriding the image tag and when we can get rid of the override?

Other than that, LGTM!

Comment (michaelimfeld) [reply charts/timed/values.yaml:225]: Added a comment. The bitnami chart uses Postgresql 11, so we probably can't get rid off it anytime soon, or am I missing something?

Comment (hairmare) [reply charts/timed/values.yaml:225]: I can't seem to find the comment :/ I should tell us why we are using PostgreSQL 12 and maybe hint at the fact that we can remove the tag override once the chart has official support for 12.x

**Pull request merged**